### DISCUSSION TITLE
Is there any way to prevent a blog from being displayed on the homepage ？

### DISCUSSION THREAD

**Discussion by Hambaobao** [adityatelange/hugo-PaperMod#234] [Q&A] [answered]
I want to make a post as my about page, but I don’t want it to appear on the homepage.

**Comment by kdkasad** [comment, marked as answer]
Set the post type to something that is not in your site's `mainSections`. See [this Hugo docs page](https://gohugo.io/functions/where/#mainsections) for more info.

**Reply by Hambaobao** [reply]
Thank you. It works.
